### :x: 1 Tests Failed:
| Tests completed | Failed | Passed | Skipped |
|---|---|---|---|
| 2163 | 1 | 2162 | 211 |
<details><summary>View the top 1 failed test(s) by shortest run time</summary>

> <pre><code class="language-python">test.integ.test_connection::test_qcc_updated_on_failed_query_with_hybrid_table</code></pre>
> <details><summary>Stack Traces | 2.12s run time</summary>
> 
> > <pre><code class="language-python">test/integ/test_connection.py:1324: in test_qcc_updated_on_failed_query_with_hybrid_table
> >     assert qcc_after_failure &gt;= qcc_after_success, (
> > E   AssertionError: QCC should not shrink after a failed query: before=2, after=0
> > E   assert 0 &gt;= 2</code></pre>
> 
> </details>

</details>

To view more test analytics, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/snowflakedb/snowflake-connector-python/tests/04-30-migrate-nodes-to-snowos)
<sub>📋 Got 3 mins? [Take this short survey](https://forms.gle/22i53Qa1CySZjA6c7) to help us improve Test Analytics.</sub>

--- a/Jenkinsfile
+++ b/Jenkinsfile
@@ -4,7 +4,7 @@ import groovy.json.JsonOutput
 
 
 timestamps {
-  node('parallelizable-c7') {
+  node('parallelizable-snowos') {
     stage('checkout') {
       scmInfo = checkout scm
       println("${scmInfo}")
@@ -113,7 +113,7 @@ timestamps {
 
 
 pipeline {
-  agent { label 'regular-memory-node' }
+  agent { label 'regular-memory-node-snowos' }
   options { timestamps() }
   environment {
     COMMIT_SHA_LONG = sh(returnStdout: true, script: "echo \$(git rev-parse " + "HEAD)").trim()

--- a/ci/test_fips.sh
+++ b/ci/test_fips.sh
@@ -20,7 +20,13 @@ CONNECTOR_DIR="$( dirname "${THIS_DIR}")"
 CONNECTOR_WHL="$(ls $CONNECTOR_DIR/dist/*cp39*manylinux2014*.whl | sort -r | head -n 1)"
 
 # fetch wiremock
-curl https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/3.11.0/wiremock-standalone-3.11.0.jar --output "${CONNECTOR_DIR}/.wiremock/wiremock-standalone.jar"
+WIREMOCK_VERSION=3.11.0
+if [[ -n "$JENKINS_HOME" ]]; then
+    WIREMOCK_BASE_URL=https://artifactory.ci1.us-west-2.aws-dev.app.snowflake.com/artifactory/development-maven-virtual
+else
+    WIREMOCK_BASE_URL=https://repo1.maven.org/maven2
+fi
+curl ${WIREMOCK_BASE_URL}/org/wiremock/wiremock-standalone/${WIREMOCK_VERSION}/wiremock-standalone-${WIREMOCK_VERSION}.jar --output "${CONNECTOR_DIR}/.wiremock/wiremock-standalone.jar"
 
 python3 -m venv fips_env
 source fips_env/bin/activate

--- a/ci/test_linux.sh
+++ b/ci/test_linux.sh
@@ -28,7 +28,13 @@ python3.10 ${THIS_DIR}/change_snowflake_test_pwd.py
 mv ${CONNECTOR_DIR}/test/parameters_jenkins.py ${CONNECTOR_DIR}/test/parameters.py
 
 # Fetch wiremock
-curl https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/3.11.0/wiremock-standalone-3.11.0.jar --output ${CONNECTOR_DIR}/.wiremock/wiremock-standalone.jar
+WIREMOCK_VERSION=3.11.0
+if [[ -n "$JENKINS_HOME" ]]; then
+    WIREMOCK_BASE_URL=https://artifactory.ci1.us-west-2.aws-dev.app.snowflake.com/artifactory/development-maven-virtual
+else
+    WIREMOCK_BASE_URL=https://repo1.maven.org/maven2
+fi
+curl ${WIREMOCK_BASE_URL}/org/wiremock/wiremock-standalone/${WIREMOCK_VERSION}/wiremock-standalone-${WIREMOCK_VERSION}.jar --output ${CONNECTOR_DIR}/.wiremock/wiremock-standalone.jar
 
 # Run tests
 cd $CONNECTOR_DIR

--- a/ci/test_rockylinux9.sh
+++ b/ci/test_rockylinux9.sh
@@ -41,7 +41,13 @@ else
 fi
 
 # Fetch wiremock
-curl https://repo1.maven.org/maven2/org/wiremock/wiremock-standalone/3.11.0/wiremock-standalone-3.11.0.jar --output ${CONNECTOR_DIR}/.wiremock/wiremock-standalone.jar
+WIREMOCK_VERSION=3.11.0
+if [[ -n "$JENKINS_HOME" ]]; then
+    WIREMOCK_BASE_URL=https://artifactory.ci1.us-west-2.aws-dev.app.snowflake.com/artifactory/development-maven-virtual
+else
+    WIREMOCK_BASE_URL=https://repo1.maven.org/maven2
+fi
+curl ${WIREMOCK_BASE_URL}/org/wiremock/wiremock-standalone/${WIREMOCK_VERSION}/wiremock-standalone-${WIREMOCK_VERSION}.jar --output ${CONNECTOR_DIR}/.wiremock/wiremock-standalone.jar
 
 # Run tests
 cd $CONNECTOR_DIR

